backend/python/app/tests/graphql/queries/test_story_query.py

```diff
@@ -0,0 +1,81 @@
+def test_stories(app, db, client):
```

Comment:
@gaoxk 
> nit: maybe this should be split into test_stories_empty and test_stories

@e-wai 
> Does it really test anything different? It's the same case logic-wise

Reply:
I think we ultimately would want different tests for an empty return/empty database state vs a filled database

Reply:
Note to self → need to guarantee data in database. Empty database was hiding problems in formatting